### PR TITLE
Tie break feature

### PR DESCRIPTION
**Pull request by ywrsusan**

### CODE REVIEW

File: pybossa/view/projects.py [hunk 1]
```diff
@@ -2994,6 +2994,19 @@ def quiz_mode(short_name):
...
+def snake_to_camel(name):
+    tags = name.split('_')
+    if len(tags) == 0:
+        return name
+    first = tags[0]
+    tags = tags[1:]
+    return first + ''.join(ele.capitalize() for ele in tags)
+
+def camel_to_snake(name):
+    import re
```
Comment (fiorda) [inline pybossa/view/projects.py]: can we import it outside?

Comment (ywrsusan) [reply pybossa/view/projects.py]: sure will do

File: pybossa/view/projects.py [hunk 2]
```diff
@@ -3037,13 +3009,12 @@ def answerfieldsconfig(short_name):
...
-            if 'answerFieldsConfig' in body:
+            if answer_fields_key in body:
                 key = answer_fields_key
-                _config = body.get('answerFieldsConfig') or {}
+                data= body.get(key) or {}
             else :
                 key = consensus_config_key
-                _config = body.get('consensusConfig') or {}
-            data = convert_to_snake(_config)
+                data = body.get(key) or {}
```
Comment (mariarivera30) [inline pybossa/view/projects.py]: This line can be after if and else since is the same line twice.

File: pybossa/view/projects.py [hunk 3]
```diff
@@ -3037,13 +3009,12 @@ def answerfieldsconfig(short_name):
     if request.method == 'POST':
         try:
             body = json.loads(request.data) or {}
-            if 'answerFieldsConfig' in body:
+            if answer_fields_key in body:
                 key = answer_fields_key
-                _config = body.get('answerFieldsConfig') or {}
+                data= body.get(key) or {}
```
Comment (mariarivera30) [inline pybossa/view/projects.py]: This line can be after if and else since is the same line twice.

File: pybossa/view/projects.py [hunk 4]
```diff
@@ -24,6 +24,7 @@
 import requests
 from StringIO import StringIO
 import six
+import re
```
Comment (fiorda) [inline pybossa/view/projects.py]: no longer needed

**Pull request merged**